core/src/main/scala/spinal/core/Area.scala

```diff
@@ -69,59 +60,48 @@ trait Area extends NameableByComponent with ContextUser with OwnableRef with Sca
           component.setPartialName(name, childNamePriority)
           OwnableRef.proposal(component, this)
         }
-      case namable: Nameable =>
+      case namable: Nameable    =>
         if (!namable.isInstanceOf[ContextUser]) {
           namable.setPartialName(name, childNamePriority)
           OwnableRef.proposal(namable, this)
-        } else if (namable.asInstanceOf[ContextUser].component == component){
+        } else if (namable.asInstanceOf[ContextUser].component == component) {
           namable.setPartialName(name, childNamePriority)
           OwnableRef.proposal(namable, this)
         } else {
-          if(component != null) for (kind <- component.children) {
-            //Allow to name a component by his io reference into the parent component
+          if (component != null) for (kind <- component.children) {
+            // Allow to name a component by his io reference into the parent component
             if (kind.reflectIo == namable) {
               kind.setPartialName(name, childNamePriority)
               OwnableRef.proposal(kind, this)
             }
           }
         }
-      case _ =>
+      case _                    =>
     }
   }
 
-  override def toString: String = (if(component != null)component.getPath() + "/"  else "") + super.toString()
+  override def toString: String = (if (component != null) component.getPath() + "/" else "") + super.toString()
 }
 
-
-
-
-/**
-  * Create an Area which can be assign to a data
+/** Create an Area which can be assign to a data
   *
-  * @example {{{
-  *     class Counter extends ImplicitArea[UInt]{
-  *        val cnt = Reg(UInt(8 bits)
-  *        ...
-  *        override def implicitValue: UInt = cnt
-  *     }
-  *     val myCounter = Counter()
-  *     io.myUInt = myCounter
-  * }}}
+  * @example
+  *   {{{ class Counter extends ImplicitArea[UInt]{ val cnt = Reg(UInt(8 bits) ... override def implicitValue: UInt = cnt } val myCounter = Counter() io.myUInt
+  * = myCounter }}}
```

Comment:
Can we somehow keep the example multi-line? I don't know how the HTML renders at the other end, but this looks not nice to look at (or even to edit) in the source code.

Reply:
+1